queries.sql

```diff
@@ -1,3 +1,90 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Find all animals whose name ends in "mon". */
+SELECT * FROM animals WHERE name LIKE '%mon%';
+/* List the name of all animals born between 2016 and 2019. */
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+/* List the name of all animals that are neutered and have less than 3 escape attempts.*/
+SELECT * FROM animals WHERE neutered = TRUE AND escape_attempt < 3;
+/* List the date of birth of all animals named either "Agumon" or "Pikachu" */
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+/* List name and escape attempts of animals that weigh more than 10.5kg */
+SELECT name, escape_attempt FROM animals WHERE weight_kg >10.5;
+/* Find all animals that are neutered. */
+SELECT * FROM animals WHERE neutered = TRUE;
+/* Find all animals not named Gabumon. */
+ SELECT * FROM animals WHERE name <> 'Gabumons';
+/* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
+ SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
### Optional
- [ ] Kindly remove one of the authors in the README file, since this project is a solo project and only one person made the commits. Also going forward, let us capitalize all our commit messages so that our repo will look professional and highlights attention to detail :smiley: